tests/testsuite/update.rs

```diff
@@ -2357,31 +2357,31 @@ fn update_breaking_spec_version_transitive() {
                 authors  =  []
 
                 [dependencies]
-                dep  =  "1.1"
+                dep  =  "2.0"
             "#,
         )
         .file("bar/src/lib.rs", "")
         .build();
 
     p.cargo("generate-lockfile").run();
 
-    Package::new("dep", "1.1.1").publish();
-    Package::new("dep", "2.0.0").publish();
+    Package::new("dep", "2.0.1").publish();
```

Comment:
What's the motivation behind 45bc28c9363d90557a4379ca56f4ca59c0d28cd7?

Reply:
In this test (`update_breaking_spec_version_transitive`) I wanted to specifically target `foo.dep` but not `bar.dep`. I can do that if `foo.dep=1.0` and `bar.dep=2.0`:

```
    p.cargo("update -Zunstable-options --breaking dep@1.0")
        .with_stderr_data(str![[r#"
[UPGRADING] dep ^1.0 -> ^3.0
[UPDATING] dep v1.0.0 -> v3.0.0
```

But not if `bar.dep=1.1`, because that makes both `foo.dep` and `bar.dep` resolve to `1.1`.